modules/ROOT/pages/using-opscenter.adoc

```diff
@@ -1,11 +1,19 @@
-= Using Ops Center on Anypoint Runtime Fabric
+= Monitor Anypoint Runtime Fabric
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes how to use the built-in Ops Center to monitor dashboards, view logs, and download debugging information within Anypoint Runtime Fabric.
+This topic describes how to use Runtime Manager or the built-in Ops Center to monitor the health of your Anypoint Runtime Fabric clusters.
```

Comment:
GLOBALLY reword to avoid self-reference to topics, sections, documents, and so forth.